app/views/study/create.scala

```diff
@@ -20,7 +20,7 @@ object create {
     views.html.site.message(
       title = trans.toStudy.txt(),
       icon = Some("4"),
-      back = data.actualFen.map(f => routes.Editor.load(f.value).url),
+      back = data.actualFen.map(f => routes.Editor.load("chess", f.value).url),
```

Comment:
do we have variant in data? There are lots of places that have "chess" as input and its unclear where this is correct or could cause issues, I suppose this will come out in testing, so wont comment on them anymore....